lib/resources/apache_conf.rb

```diff
@@ -85,6 +85,12 @@ def read_content
           assignment_regex: /^\s*(\S+)\s+((?=.*\s+$).*?|.*)\s*$/,
           multiple_values: true,
         ).params
+
+        # Capture any characters between quotes that are not escaped in values
+        params.values.map! do |x|
```

Comment:
Could we not use `x` and `y` here?

Reply:
Sure can, I changed it to `value` and `sub_value`.

Let me know if you have better names...I'm bad with naming things.